main.rb

```diff
@@ -0,0 +1,103 @@
+require_relative 'app'
+require_relative 'book'
+require_relative 'person'
+require_relative 'rental'
+
+def display_menu
+  puts 'What would you like to do?'
+  puts '1. List all books'
+  puts '2. List all people'
+  puts '3. Create a person'
+  puts '4. Create a book'
+  puts '5. Create a rental'
+  puts '6. List all rentals for a given person id'
+  puts '0. Quit'
```

Comment:
- [ ] Please add the welcoming title before options and update the last option to '7' as in the video, also when '7' is selected kindly return this message:
![image](https://user-images.githubusercontent.com/91486791/232101228-8dc84cc6-678c-4b79-b64b-169ea68c82fe.png)


> ![image](https://user-images.githubusercontent.com/91486791/232096574-65ffd183-44b5-46a3-8dd1-77a1938acda6.png)
>![image](https://user-images.githubusercontent.com/91486791/232096888-60aae1d7-d8e2-44eb-8e79-0a3a11b61cd8.png)

Reply:
Done, thank you